<?xml version="1.0" encoding="utf-8"?>
<launch>
    <!-- Console launch prefix -->
    <include file="$(find robot)/launch/D435.launch" />

    <arg name="launch_prefix" default=""/>

    <!-- Config and weights folder. -->
    <arg name="yolo_weights_path"          default="$(find darknet_ros)/model"/>
    <arg name="yolo_config_path"           default="$(find darknet_ros)/cfg"/>

    <!-- Load parameters -->
    <rosparam command="load" ns="darknet_ros" file="$(find darknet_ros)/params/ros.yaml"/>
    <!--<rosparam command="load" ns="darknet_ros" file="$(find darknet_ros)/params/yolov3-tiny-home.yaml"/>-->
    <rosparam command="load" ns="darknet_ros" file="$(find darknet_ros)/params/yolov3-tiny-ouquanlin.yaml"/>
    <!--<rosparam command="load" ns="darknet_ros" file="$(find darknet_ros)/params/yolov3-gpsr.yaml"/>-->
    <!-- Start darknet and ros wrapper -->
    <node pkg="darknet_ros" type="darknet_ros" name="darknet_ros" output="screen" launch-prefix="$(arg launch_prefix)">
        <param name="weights_path"          value="$(arg yolo_weights_path)" />
        <param name="config_path"           value="$(arg yolo_config_path)" />
    </node>

</launch>
